lib/skate_web/channels/notifications_channel.ex

```diff
@@ -15,8 +12,10 @@ defmodule SkateWeb.NotificationsChannel do
     end
   end
 
-  @impl true
-  def join("notifications", _message, socket) do
+  use SkateWeb.AuthenticatedChannel
```

Comment:
**nitpick:** I guess this isn't a strict requirement since it passes CI, but I think it would be easier to read this file if all of the `use` / `import` / `alias` statements are grouped together up top.